### DISCUSSION TITLE
Samples on Linux?

### DISCUSSION THREAD

**Discussion by ib00** [jrouwe/JoltPhysics#120] [Q&A] [answered]
Has anybody managed to compile samples on linux?

**Comment by jrouwe** [comment, marked as answer]
You can't compile them on Linux. Only HelloWorld, PerformanceTest and the UnitTests run on Linux currently. The samples require DX12. I'm planning to make a Vulkan based renderer in the future.

**Reply by ib00** [reply]
Thank you! Fantastic project!

Another possibility would be to use 'sokol_gfx' (which has OpenGL, DX, Metal, WebGL backends).

**Reply by jrouwe** [reply]
The samples run natively on Linux now

**Reply by ib00** [reply]
Thank you!
